dgraph/cmd/zero/raft.go

```diff
@@ -311,7 +311,7 @@ func (n *node) applySnapshot(snap *pb.ZeroSnapshot) error {
 	}
 	n.server.orc.purgeBelow(snap.CheckpointTs)
 
-	data, err := snap.Marshal()
+	data, err := snap.GetState().Marshal()
```

Comment:
Add a comment here `We only need to store the membership state in the snapshot`

Reply:
Done.